create/application.go

```diff
@@ -29,11 +31,11 @@ type applicationCmd struct {
 	Wait        bool              `default:"true" help:"Wait until application is fully created."`
 	WaitTimeout time.Duration     `default:"15m" help:"Duration to wait for application getting ready. Only relevant if wait is set."`
 	Git         gitConfig         `embed:"" prefix:"git-"`
-	Size        string            `default:"micro" help:"Size of the app."`
-	Port        int32             `default:"8080" help:"Port the app is listening on."`
-	Replicas    int32             `default:"1" help:"Amount of replicas of the running app."`
+	Size        *string           `help:"Size of the app (defaults to \"${app_default_size}\")."`
```

Comment:
I just noticed something when looking at the help of this branch. Whenever a flag is a pointer, the help will look like this: `--size=SIZE` instead of `--size=STRING`. This is IMO especially problematic for bools, such as `--basic-auth=BASIC-AUTH`. It's rather confusing to know what I now need to supply here. Should we maybe use the `placeholder` tag for these? So e.g. `placeholder="micro"`, so the help shows a real example like `--size=micro` (which is also like it was with the default before).

But sadly it looks like the placeholder does not support variables.

Reply:
Good catch. I will have a look. As I tested with a different approach yesterday I might have an idea.